### ISSUE TITLE
Snippet body used as description when none is provided

### ISSUE THREAD

**Issue by dimateos** [microsoft/vscode#181115]
For tiny snippets I usually end up writing the same body as the description as it provides the perfect info (plus also the uncut positional args as I mention in https://github.com/microsoft/vscode/issues/181112).

This is nice because you have a tiny preview in the insert snippet menu, PLUS you also get fuzzy match agains part of the body:
![image](https://user-images.githubusercontent.com/26868611/235124504-0cd73e6f-6313-40b8-9174-cb181edd2ace.png)

With no description is hard to remember sometimes:
![image](https://user-images.githubusercontent.com/26868611/235124428-66ab6d10-881b-427a-a9e7-a057d1a4510e.png)

So it would be nice to have an option to toggle this behavior:
* Just show a preview of the snippet in the insert snippet window, like is shown in the snippet suggestion documentation popup (see images and more context in https://github.com/microsoft/vscode/issues/181112)
* Actually take the body string as description whenever there is no description. Would achieve fuxxy search. Could limit to single string bodies or just concatenate big bodies, but a large body could e a problem.

**Comment by TheSylvester** [comment]
Hi everyone!  I'm a relatively new dev trying to make my first contrib to a project and this seems to be the only "good first issue" that seems easiest, so I've been trying to investigate how to work on it.

If anyone has any advice on where to start looking, I'm all ears 👂😆

**Comment by jrieken** [comment]
@TheSylvester this is your code pointer: https://github.com/microsoft/vscode/blob/193bec8ce8179d2340823b9a29e7bc130548357f/src/vs/workbench/contrib/snippets/browser/snippetPicker.ts#L39

**Comment by TheSylvester** [comment]
> 

Thank you!  Coincidentally I was examining this exact piece when you posted the comment.  I implemented the functionality requested in my own code.  Now I just need to turn it into a toggleable option in prefs 😉

Now that I've finally got past the initial shock of the first large code base I've read into, I'll be looking at other PRs to examine how they added preferences and go from there.  Hopefully I will have a PR soon 🤞
